packages/powersync_attachments_stream/lib/src/attachment_service.dart

```diff
@@ -0,0 +1,13 @@
+import 'attachment_state.dart';
+import 'attachment_context.dart';
+
+abstract class AttachmentService {
+  Future<void> init();
+  Future<void> close();
+
+  Stream<List<Attachment>> watchActiveAttachments();
```

Comment:
Do you think it makes sense to move of the methods related to fetching attachments to the `AttachmentContext` class like it's done in Kotlin?